tests/integration/data/testMigrate.json

```diff
@@ -0,0 +1,23 @@
+{
+  "migration": {
+    "name": "myMigrationTest",
+    "os": "linux",
+    "source": {
+      "host": "10.0.0.211",
+      "ssh-port": 22,
+      "user": "root",
+      "password": "password"
+    },
+    "destination": {
+      "platform": "VMware vCenter",
+      "format": "VMware vCenter",
+      "displayName": "weasel-jerome-vcenter2",
+      "esxHost": "esx4dev.hq.usharesoft.com",
+      "datastore": "esx4dev_data1_secure",
+      "network": "VM Network",
+      "account": {
+        "name": "account_name"
+      }
+    }
+  }
+}
```

Comment:
this files is stored in the resources of ITs, but used in UTs

Reply:
I deleted this file